### PR TITLE
Added Rodrigues to the JS embindgen.py and a unittest in test_calib3d…

### LINKED ISSUE #15584: Build Rodrigues in Javascript
##### System information (version)

- OpenCV => 4.1.1
- Operating System / Platform => Windows 64 Bit with Windows subsystem for Linux
- Compiler => Emscripten

##### Detailed description

I would like the Rodrigues function to be built in Javascript.

##### Steps to reproduce
Insert the following code into the bottom of test_calib3d.js, compile OpenCV.js and the following test fails.

```
QUnit.test('Rodrigues', async function(assert) {
const rvec0 = cv.matFromArray(1, 3, cv.CV_64F, [1,1,1])
let rMat0 = new cv.Mat()
let rvec1 = new cv.Mat()

cv.Rodrigues(rvec0, rMat0)

assert.ok(rMat0.data64F.length == 9)
assert.ok(0.23 > rMat0.data64F[0] > 0.22)

cv.Rodrigues(rMat0, rvec1)

assert.ok(rvec1.data64F.length == 3)
assert.ok(1.01 > rvec1.data64F[0] > 0.9)
});
```

All tests should pass.

### PR DESCRIPTION
**Pull request by frastlin**
I did something wrong and PR #15585 closed. This PR is on 3.4 and has the changes.

Resolves #15584

### This pullrequest changes

Adds the Rodrigues function to the Calib3D module and adds a unittest in the js/tests folder.

```
force_builders=Custom
buildworker:Custom=linux-1,linux-2,linux-4
docker_image:Docs=docs-js
docker_image:Custom=javascript
```

### CODE REVIEW

File: modules/js/src/embindgen.py [hunk 1]
```diff
@@ -155,7 +183,7 @@ def makeWhiteList(module_list):
                 wl[k] = m[k]
     return wl
 
-white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, calib3d])
+white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, photo, aruco, calib3d])
```
Comment (alalek) [inline modules/js/src/embindgen.py]: Please remove unrelated changes (they are from the master branch only)

**Comment by frastlin** [comment]
OK, I fixed it, and pushed to my PR branch on my fork, will it update here like it says, or do I need to do anything else?
